docs/release-notes/1596.dev.md

```diff
@@ -0,0 +1,2 @@
+update the CI to use `uv` {user}`Intron7`
+create new `cupy` install options for cuda 11 & 12 {user}`Intron7`
```

Comment:
```suggestion
create new `cupy` installation options for cuda 11 & 12 called `cu11` and `cu12` {user}`Intron7`
```

Reply:
We shouldn't be putting CI changes in fragments and should only have one note per fragment.